app/src/main/java/com/imashnake/animite/features/searchbar/SearchFrontDrop.kt

```diff
@@ -62,7 +62,7 @@ import com.imashnake.animite.R as Res
 )
 @Composable
 fun SearchFrontDrop(
-    modifier: Modifier,
+    modifier: Modifier = Modifier,
```

Comment:
Default parameters should be last in the list, bring `navController` up (or remove it as per my last comment

Reply:
o7